lesson6.html

```diff
@@ -0,0 +1,130 @@
+    <title>Document</title>
+    <style>
+        #oldArea {
+            margin: 0 60px 30px 60px;
+        }
+
+        #btnTransfer {
+            padding: 30px;
+            margin: 0 60px 0 60px;
+        }
+
+        #newTrans {
+            margin: 60px 60px 0 60px;
+        }
+    </style>
+</head>
+
+<body>
+
+    <div id="cars">
+
+
+        <div>
+            <h1>Format Code</h1>
+            <div id="oldArea">
+                <textarea id="oldValue" type="text" class="form-control"></textarea>
+            </div>
+            <div id="btnTransfer">
+                <button type="button" class="btn-btn-primary" onclick="transfer();">
+                    Transfer
+                </button>
+            </div>
+            <div id="oldArea">
+                <textarea id="newValue" type="text" class="form-control"></textarea>
+            </div>
+        </div>
+
+        <div>
+            <h1>Balance</h1>
+            <div id="oldArea">
+                <textarea id="orginText" type="text" class="form-control"></textarea>
+            </div>
+            <div id="btnTransfer">
+                <button type="button" class="btn-btn-primary" id="balance" onclick="balance()">
+                    Transfer
+                </button>
+            </div>
+            <div id="oldArea">
+                <textarea id="newText" type="text" class="form-control"></textarea>
+            </div>
+        </div>
+    </div>
+    <script>
+
+
+        var textOld = document.getElementById("oldValue");
+        var textNew = document.getElementById("newValue");
+        var origintext = document.getElementById("orginText");
+        var newM = document.getElementById("newText");
+
+        // const str = 'let a  =12;';
+
+        // const result = str.replace("=", " = ");
+        // console.log(result); // 👉️ "a p p l e"
+
+        function transfer() {
+            textNew.value = textOld.value.split(' ').filter(s => s).join(' ');
+            console.log(textNew.value);
+            textNew.value = textNew.value.trim();
+            console.log(textNew.value);
+
+            // textNew.value = textNew.value.split("=");
+            // console.log(textNew.value);
+            // textNew.value = textNew.value.replace(",", "=");
+            // console.log(textNew.value);
+            textNew.value = textNew.value.replace("=", " = ");
+            console.log(textNew.value);
+            textNew.value = textNew.value.replace(" ;", ";");
+            console.log(textNew.value);
+            textNew.value = textNew.value.replace("=", " = ");
+            textNew.value = textNew.value.split(' ').filter(s => s).join(' ');
+            console.log(textNew.value);
+        }
+
+
+
+        function checkboo(s) {
+
+
+            let par = {
+                ")": "(",
+                "}": "{",
+                "]": "["
+            };
+            let s1 = [];
+            let i;
+            for (i = 0; i < s.length; i++) {
```

Comment:
s.length nên để ra ngoài vòng for e ơi
https://www.w3schools.com/js/js_performance.asp